class/defaults.yml

```diff
@@ -165,48 +186,43 @@ parameters:
         - kube-apiserver-slos-basic
       customAnnotations: {}
       patchRules:
-        # rules patched in '*' will be applied regardless of the value of
-        # parameter `manifests_version`.
-        '*':
-          HighOverallControlPlaneMemory:
-            annotations:
-              description: |
-                The overall memory usage is high.
-                kube-apiserver and etcd might be slow to respond.
-                To fix this, increase memory of the control plane nodes.
+        HighOverallControlPlaneMemory:
```

Comment:
This restructuring is a breaking change!

(IMO that's ok, but we'll need to update our Commodore defaults accordingly for the rollout)